components/modal/index.jsx

```diff
@@ -177,6 +177,7 @@ const defaultProps = {
 	},
 	align: 'center',
 	ariaHideApp: true,
+	size: 'small',
```

Comment:
I think adding a default value here would be a breaking change.

Reply:
Oops, I forgot about that. I will update it.

Reply:
I have updated it :+1: